desktop/src/args.rs

```diff
@@ -25,6 +28,23 @@ pub(crate) struct Args {
         help = "Path to the ROM that should be loaded into the emulator.",
     )]
     pub(crate) path_to_rom: PathBuf,
+
+    #[structopt(
+        long = "--breakpoints",
+        parse(try_from_str = "parse_breakpoint"),
+        requires = "debug",
+        help = "Breakpoint that is added to the debugger at the very beginning. Breakpoints are \
```

Comment:
It is not clear, if the user can pass one or many breakpoints (and if many, how they have to be separated (e.g. `0x42, 0x21` or `0x42;0x21` or `0x42 0x21` etc.)).